extensions/data-workspace/src/dataworkspace.d.ts

```diff
@@ -55,8 +55,9 @@ declare module 'dataworkspace' {
 		 *
 		 * @param name Create a project
 		 * @param location the parent directory of the project
+		 * @param projectTypeId the identifier of the selected project type
 		 */
-		createProject(name: string, location: vscode.Uri): Promise<vscode.Uri>;
+		createProject(name: string, location: vscode.Uri, projectTypeId: string): Promise<vscode.Uri>;
```

Comment:
Call it ProjectTemplateType to be consistent with Raden's mocks may be... like
Project types - sqlproj
Template types - simple db project, Edge project etc...
what do you think?

Reply:
I like `projectTemplateType` or `projectTemplateId` better than what I used, but wanted to keep the name consistent throughout the data workspaces extension.  Think it's worth 1) keeping it consistent, and 2) doing that change (which would touch a lot more) as part of this port?

My gut says to keep this PR small (because it'll be the port), then fixing up the variable name across the entire extension in a different PR after release.

Reply:
(@alanrenmsft @kisantia)

Reply:
yeah makes sense to change this in another PR after this release

Reply:
>  doing that change (which would touch a lot more) 

Didnt catch this part sorry - I thought since this is new argument the changes are constained only. I am okay with leaving as is then. your call.